package.json

```diff
@@ -21,6 +21,7 @@
     "@types/babel__template": "^7.0.2",
     "@types/node": "~14.14.45",
     "@types/which": "^3.0.0",
+    "@types/ws": "8.5.1",
```

Comment:
Looks like the newer 8.5.5 is causing failures due to a breaking change (a type became non-generic while it was before)